src/TetherComponent.jsx

```diff
@@ -96,6 +96,15 @@ class TetherComponent extends Component {
       this._createContainer();
     }
 
+    // Verify if className props have changed
+    if (prevProps.className !== this.props.className) {
+      const prevClasses = (prevProps.className || '').split(' ');
+      const currClasses = (this.props.className || '').split(' ');
+
+      this._elementParentNode.classList.remove(...prevClasses);
```

Comment:
This step could probably be more optimized in only adding/removing classes that do not overlap. This is just simpler to read though.